provides aligning with Zipkin b3-propagation (#304)
rsocket/rsocket: Extensions/Tracing-Zipkin.md

```

* **Flags**: (8 bits)

>   * (**B**)ody Set: Tracing payload is set. 

Reviewer: ```suggestion
  * (**I**)Ds Set: When zero, the metadata only includes sampling information and no IDs
    * For example, a health check might set only the (**N**)ot Sampled flag without generating IDs
```

  * (**D**)ebug: Tracing payload should be force traced.
  * (**S**)ample: Tracing payload should be accepted for tracing. (Ignored when D flag is set.)

>   * (**R**)eject: Tracing payload should not be sampled. (Ignored when S flag or D flag is set.)

Reviewer: I see the reject notion is sticking with you vs sampling set. If you do that, I think reject is still a bad word to use in flags used for different things. We never use it in practice, we only used 'Reject' in the README to choose a word besides Sampled, and in that case we pair it with Accept. Reject doesn't pair with Sampled and is hence more jargon.

```suggestion
  * (**N**)ot Sampled: Tracing payload should not be sampled. (Ignored when S flag or D flag is set.)
```

I looked and in practice some say unsampled a little more say not sampled.

I think that this is important enough to pick a good name and not blindly follow the text description in the README as I've mentioned that was not trying to pick flag names and if anything we should pick names that are similar to ones in use to cause least confusion, right?

Once we settle on this point I'd like to ping some other zipkin folks as this could well be inherited into our spec.

* **Flags**: (8 bits)
  * (**T**)race Id Size: Unset indicates that the Trace Id is 64-bit. Set indicates that the Trace Id is 128-bit.
  * (**P**)arent Span Id: Tracing payload contains a parent span id.
  * (**R**)eject: Tracing payload should not be sampled. (Ignored when S flag or D flag is set.)

>   * (**S**)ampl: Tracing payload should be accepted for tracing. (Ignored when D flag is set.)

Reviewer 2: ```suggestion
  * (**S**)ample: Tracing payload should be accepted for tracing. (Ignored when D flag is set.)
```
